CHANGES.rst

```diff
@@ -13,6 +13,35 @@ Changelog
 
 .. towncrier release notes start
 
+3.13.2 (2021-06-23)
+
+Bugfixes
+--------
+
+- Taught sync to process modulemd before packages so is_modular can be known.
+  (backported from #8952)
```

Comment:
It's kind of weird to have these written as backports- I didn't really intend that, it's just what ended up happening due to the process.  

Since they are being fixed prior to 3.14 it does make some sense to drop them from the 3.14 changelog.  It's weird but "fine" I think.